docs/README-agent-install.md

```diff
@@ -206,13 +216,176 @@ search mycompany.com
 EOF
 ~~~
 
+## <a name="env-gs">Puppet Agent Environment: open agent container</a>
+
+This section is only necessary if Puppet will run from the `open agent container`.
+
+#### Set Up NX-OS
+
+Download the `OAC` `oac.1.1.0.ova` file.
+
+* For Cisco Nexus 7xxx devices: [Download Link](https://software.cisco.com/download/release.html?i=!y&mdfid=283748960&softwareid=282088129&release=7.3%280%29D1%281%29&os=)
+
+* For Cisco Nexus 56xx and 60xx devices: [Download Link]**TODO**()
```

Comment:
I suggest a table for these downloads:


| Platform | OAC Download Link |
|:---|:---|
| Nexus N7xxx | [Download Link](https://software.cisco.com/download/release.html?i=!y&mdfid=283748960&softwareid=282088129&release=7.3%280%29D1%281%29&os=) |
| Nexus N56xx <br> Nexus N60xx | [Download Link]**TODO**() |